src/NewSubscription.m

```diff
@@ -202,6 +202,7 @@ -(IBAction)doSubscribe:(id)sender
 	// Validate the subscription, possibly replacing the feedURLString with a real one if
 	// it originally pointed to a web page.
 	rssFeedURL = [subscriptionModel verifiedFeedURLFromURL:rssFeedURL];
+	NSAssert(rssFeedURL != nil, @"No valid URL verified to attempt subscription !");
```

Comment:
Try this valid feed: https://feeds.feedburner.com/UnifiedArsTechnicaFeed?format=xml
It's always set to `nil` for me and therefore triggers the assertion failure.